docs/user_guide.md

```diff
@@ -456,6 +456,12 @@ Each line in the skip file begins with a `-` or a `+`, followed by a path glob
 pattern. `-` means that if a file matches a pattern it should **not** be
 checked, `+` means that it should be.
 
+Absolute directory paths should start with `/`.
+Relative directory paths should start with `*`.
+Path parts should start and end with `*`.
+
+To skip everything use the `-*` mark (Watch out for the order!).
```

Comment:
Maybe **Watch out for the order!** sentence can be highlighted or to be bold because this is an important information.